swap/htlc.go

```diff
@@ -108,17 +108,26 @@ type Htlc struct {
 }
 
 var (
-	quoteKey [33]byte
-
+	dummyPubKey = [33]byte{
```

Comment:
Oh sorry, I don't know why I made that comment. It's actually used for the pre-constructed quote htlcs. We need an actual pubkey there since it's parsed when the htlc is constructed. Sorry again for the confusion.

Reply:
done